### PR TITLE
Build against a newer gdal, bump version

### PR DESCRIPTION
**Pull request by dpryan79**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dpryan79** [comment]
@conda-forge-admin please rerender

**Comment by dpryan79** [comment]
This is related to https://github.com/bioconda/bioconda-recipes/pull/15134 wherein `r-rgdal` was being built against a libgdal version that was pulling in an incompatible version of `proj4`.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,11 +30,13 @@ requirements:
   host:
     - r-base
     - r-sp >=1.1_0
-    - libgdal
+    - gdal >=1.11.4,<=2.5.0
+    - proj4 >=4.8.0,<6
```
Comment (ocefpaf) [inline recipe/meta.yaml]: You should not pin it like this b/c you may never get a solvable solution. Let the central pinning handle it.

**Comment by ocefpaf** [comment]
Done in #21

**Pull request closed without merging**